hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.js

```diff
@@ -18,16 +19,34 @@ import {
     orderStatus,
     members,
     teamCode,
+    productInformation,
 } from "testing/mockData";
 import { hackathonName } from "constants.js";
 
 const Dashboard = () => {
     const reportIncident = (id) => {
         alert("Reports incident for component of id " + id);
     };
+
+    const addToCart = () => {
+        alert("add to cart");
+    };
+
+    const [mobileOpen, setMobileOpen] = React.useState(true);
```

Comment:
If you want to keep this around until we can actually hook it up to redux and make the info buttons open it, make the default `false`.